test/e2e/privilegesmgmt/ssr.go

```diff
@@ -45,84 +45,91 @@ func SSRTest() {
 	BeforeEach(func() {
 		flag.Parse()
 		if VeleroCfg.InstallVelero {
-			Expect(VeleroInstall(context.Background(), &VeleroCfg, "", false)).To(Succeed())
+			err = VeleroInstall(context.Background(), &VeleroCfg, "", false)
+			Expect(err).To(Succeed(), "Failed to install velero with error %v", err)
 		}
 	})
 
 	AfterEach(func() {
 		if VeleroCfg.InstallVelero {
-			Expect(VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)).To(Succeed())
+			err = VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)
+			Expect(err).To(Succeed(), "Failed to install velero with error %v", err)
 		}
 	})
 
 	It(fmt.Sprintf("Should create an ssr object in the %s namespace and later removed by controller", VeleroCfg.VeleroNamespace), func() {
-		defer DeleteNamespace(context.TODO(), client, testNS, false)
+		defer func() {
+			err = DeleteNamespace(context.TODO(), client, testNS, false)
+			Expect(err).To(Succeed(), "Failed to delete the namespace %s with error %v", testNS, err)
+		}()
 		ctx, _ := context.WithTimeout(context.Background(), time.Duration(time.Minute*10))
-		By(fmt.Sprintf("Create %s namespace", testNS))
-		Expect(CreateNamespace(ctx, client, testNS)).To(Succeed(),
-			fmt.Sprintf("Failed to create %s namespace", testNS))
+		By(fmt.Sprintf("Create %s namespace", testNS), func() {
+			err = CreateNamespace(ctx, client, testNS)
+			Expect(err).To(Succeed(), "Failed to create %s namespace with error %v", testNS, err)
+		})
+
+		By(fmt.Sprintf("Get version in %s namespace", testNS), func() {
+			cmd := []string{"version", "-n", testNS}
+			err = VeleroCmdExec(context.Background(), VeleroCfg.VeleroCLI, cmd)
+			Expect(err).To(Succeed(), "Failed to create an ssr object in the %s namespace with error %v", testNS, err)
```

Comment:
Log content is not right.

Reply:
if we execute command `velero version -n testNS`, which will create an ssr cr in testNS namespace by the controller